tensorflow_datasets/structured/movielens.py

```diff
@@ -78,98 +105,78 @@ class MovieLens(tfds.core.GeneratorBasedBuilder):
       tfds.core.BuilderConfig(
           name='25m',
           description=textwrap.dedent("""\
-              This dataset contains 25000095 ratings across 62423 movies.
-              These data were created by 162541 users between January 09, 1995
-              and November 21, 2019.
-              Ratings are in half-star increments.
-              Each user has rated at least 20 movies.
-              This dataset is the latest stable version
-              of the MovieLens dataset.
-              This dataset does not include demographic data.
-              The movies data and ratings data are joined on "movieId".
-              Genres are converted into sequence of labels.
-              "(no genres listed)" is considered a label itself.
-              This dataset was generated on November 21, 2019."""),
+              This dataset contains 25,000,095 ratings across 62,423 movies.
```

Comment:
More writing nits: this paragraph feels a little unnatural. What about rephrashing it to sound a little bit smoother? For example, there is a lot repetition of "This dataset"; we could perhaps avoid it by writing something like:

This dataset contains 25,000,095 ratings across 62,423 movies, created by 162,541 users between January 1996 and November 2019. It is the latest stable version of the MovieLens dataset, generated in November 2019. 

The dataset contains genre data (converted into a sequence of labels), but does not contain demographic data. Each user has rated at least 20 movies, and ratings are given in half-star increments.

Reply:
Good point. I have revised the documentations in dfe803aa3e5f9ddba2d9e26214c612f095ea1b2a. Could you take a look and let me know if it is better now?

Reply:
Yes, thank you!